stubs/swiftclient/exceptions.pyi

```diff
@@ -1,2 +1,8 @@
+from typing import Optional
+
+
 class ClientException(Exception):
+    do_not_retry: Optional[bool]
```

Comment:
This might deserve a comment, since the underlying library doesn't provide this attribute, we're "mixing it in" dynamically later.